rest-server/admin/src/main/scala/com/actionml/admin/MongoAdministrator.scala

```diff
@@ -106,7 +106,7 @@ class MongoAdministrator extends Administrator with JsonSupport {
     parseAndValidate[GenericEngineParams](json).andThen { params =>
       engines.get(params.engineId).map { existingEngine =>
         logger.trace(s"Re-initializing engine for resource-id: ${params.engineId} with new params $json")
-        enginesCollection.saveOne(EngineMetadata(params.engineId, params.engineFactory, json))
+        enginesCollection.saveOneById(existingEngine.engineId, EngineMetadata(params.engineId, params.engineFactory, json))
```

Comment:
@andreyshikov Are we deprecating `saveOne`? That would mean delete it from DAO. That would be ok with me.

Reply:
no, I changed savedOne syntax and now it looks good (it's update)